core/arch/arm/include/ffa.h

```diff
@@ -9,7 +9,9 @@
 
 /* This is based on the FF-A 1.0 EAC specification */
 
+#if defined(ARM64) || defined(ARM32)
```

Comment:
Unrelated change?

Reply:
No, it wasn't. However I changed it to be more clear.
For the spmc tests in xtest we need the ffa.h file, the ffa.h file needs the smccc.h. So I changed it to copy both files when the SPMC_TEST are selected.